web/src/GameScreen.js

```diff
@@ -1,47 +1,78 @@
-import React, { useContext, useState } from 'react';	
+import React, { useContext, useState, useEffect } from 'react';	
 import { Context } from './context';
-import {Grid, Paper, Typography, Button, TextField} from '@material-ui/core';
+import { Grid, Paper, Typography, Button, TextField } from '@material-ui/core';
 
 function GameScreen(classes) {
-  const [context, setContext] = useContext(Context)
-  const [displayWord, setDisplayWord] = useState()
-  const [word, setWord] = useState('')
-  const [char, setChar] = useState('')
-  console.log(`[UI GameScreen]: load`)
+  const [context, setContext] = useContext(Context);
+  const [displayWord, setDisplayWord] = useState('');
+  const [word, setWord] = useState('');
+  const [char, setChar] = useState('');
+  const [usedChars, setUsedChars] = useState([]);
+  const [misses, setMisses] = useState(0);
+  const [maxMisses, setMaxMisses] = useState(0);
+  const [isDisabled, setIsDisabled] = useState(false);
+  console.log(`[UI GameScreen]: load`);
+
+  useEffect(() => {
```

Comment:
why not just make this anonymous method async? or does it not work?

Reply:
```javascript
Effect callbacks are synchronous to prevent race conditions. Put the async function inside:

useEffect(() => {
  async function fetchData() {
    // You can await here
    const response = await MyAPI.getData(someId);
    // ...
  }
  fetchData();
}, [someId]); // Or [] if effect doesn't need props or state

Learn more about data fetching with Hooks: https://fb.me/react-hooks-data-fetching
```